pkg/util/disk/usage_unix.go

```diff
@@ -1,3 +1,5 @@
+// +build !windows
```

Comment:
`FreeSpace` on Linux/macOS relies on [statfs](https://man7.org/linux/man-pages/man2/statfs.2.html) bfree (free blocks in filesystem), consider use of bavail (free blocks available _to unprivileged user_).